rdt-open-reader/src/covidIndonesia/assets/json.form/patient-registration-form-original.json

```diff
@@ -659,7 +659,7 @@
           }
         },
         "duration": {
-          "label": "Test Date"
+          "label": "Check Day"
```

Comment:
Was this the issue? Does changing the text fix the problem?

Reply:
As per my understanding, it was a translation issue which I asked from ellya https://github.com/onaio/rdt-standard/issues/810#issuecomment-788885806 and she was replied me https://github.com/onaio/rdt-standard/issues/810#issuecomment-789625377.
Correct me if have done something wrong.